services/dis-console-ui/Makefile

```
@@ -0,0 +1,198 @@
+
+.PHONY: image-build
+image-build: ## Build the container image (Bun build + Bun-served) with $(CONTAINER_TOOL).
+	$(CONTAINER_TOOL) build -t $(IMG) .
+
+.PHONY: serve
+serve: image-build ## Build + run the image on $(E2E_URL) (only needs $(CONTAINER_TOOL); no Kind, no host toolchain). Ctrl-C to stop.
+	@echo "Serving $(E2E_URL) (Ctrl-C to stop)..."
+	$(CONTAINER_TOOL) run --rm -it -p $(E2E_PORT):8080 $(IMG)
+
+.PHONY: kind-create
+kind-create: ## Create the Kind cluster if it does not exist (writes a dedicated kubeconfig).
```

checkmake (0.3.2)

[warning] Target body for "kind-create" exceeds allowed length of 5 lines (8).

(maxbodylength)